package.json

```diff
@@ -10,20 +10,21 @@
   "license": "MIT",
   "main": "./index.js",
   "scripts": {
-    "coveralls": "nyc report --reporter=text-lcov | coveralls",
-    "coverage": "nyc report --reporter=html",
-    "test": "./node_modules/.bin/jshint * && nyc ./node_modules/.bin/_mocha ./test/*.js ./test/commands/*.js ./test/parser/*.js --timeout=8000",
-    "jshint": "./node_modules/.bin/jshint *"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "test": "nyc ./node_modules/.bin/_mocha ./test/*.js ./test/commands/*.js ./test/parser/*.js --timeout=8000",
+    "pretest": "optional-dev-dependency hiredis",
+    "posttest": "jshint ."
   },
   "devDependencies": {
     "coveralls": "^2.11.2",
-    "hiredis": "^0.4.1",
```

Comment:
node_hiredis states that it [supports windows since v.0.3.0] (https://github.com/redis/hiredis-node/blame/master/README.md#L77)? Do you have any idea why it does not work?